packages/get-platform/jest.config.js

```diff
@@ -4,9 +4,22 @@ module.exports = {
   },
   transformIgnorePatterns: [],
   testEnvironment: 'node',
+  testMatch: ['**/src/**/*.test.ts'],
   collectCoverage: process.env.CI ? true : false,
   coverageReporters: ['clover'],
   coverageDirectory: 'src/__tests__/coverage',
   collectCoverageFrom: ['src/**/*.ts', '!**/__tests__/**/*'],
-  testMatch: ['**/src/__tests__/**/*.test.ts'],
+  snapshotSerializers: ['./src/test-utils/jestSnapshotSerializer'],
+  reporters: [
+    'default',
+    [
+      'jest-junit',
```

Comment:
You'll need to add `jest-junit` as a dev dependency (like done in the other packages)